README.md

```diff
@@ -5,15 +5,25 @@
 * Go 1.15 or better
 
 ## Download & Install
+###With Brew [Recommended]:
```

Comment:
Does it make sense to add a blank line between headers? And also add a space between `#` and `W`? I think it would be better to remove the colon at the end.

Would it also make sense to remove `[Recommended]`?

Reply:
Yes makes sense, also maybe we can follow Jiri's advice on how to direct user group to brew installation. Will remove it for the time being